rest/handler.go

```diff
@@ -127,6 +127,11 @@ func (h *handler) invoke(method handlerMethod) error {
 		}
 	}
 
+	h.logRequestLine()
+	if base.EnableLogHTTPBodies {
+		h.logRequestBody()
```

Comment:
logRequestBody is based on h.requestBody - it looks like it needs to be after the switch block below.